internal/test/providers/cloudprovider.go

```diff
@@ -0,0 +1,33 @@
+package providers
+
+import (
+	"fmt"
+
+	"github.com/openshift/api/config/v1"
+	mapi "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
+	oc "github.com/openshift/windows-machine-config-bootstrapper/internal/test/clusterinfo"
+	awsProvider "github.com/openshift/windows-machine-config-bootstrapper/internal/test/providers/aws"
+	"github.com/pkg/errors"
+)
+
+type CloudProvider interface {
```

Comment:
I just noticed this stutters, i.e. `providers.CloudProvider`. Did you do this to maintain parity with WMCO? If not, `Cloud` should suffice here.

Reply:
Actually this is coming directly from the WMCO repo, kept this as it is on purpose. Should i change it to cloud ?

Reply:
No, given you are doing it to maintain parity with WMCO.